### ISSUE TITLE
Adding Edge browser details to the Wiki

### ISSUE THREAD

**Issue by JoMangee** [OpenUserJs/OpenUserJS.org#1046]
I've taken the liberty of screenshotting my Tampermonkey install in Windows 10's Edge browser and edited the wiki page locally, adding details of Edge install to a Tampermonkey-for-Edge.md

![tampermonkey-edge](https://cloud.githubusercontent.com/assets/17394/20637655/35ffa608-b3f3-11e6-839b-a1162cf02879.png)

Will you accept a Pull Request for this?

**Comment by Martii** [comment]
> Will you accept a Pull Request for this?

As long as the documentation is neutral e.g. not showing your user icon in the screen shots, not promoting any particular script other than the establishing owners *(Usually it's the GM_config Unit Test since that's about as neutral as we can get.)* ... only promoting OUJS site other than where to get TM for Edge, hyperlinks as described in [the main GH wiki page](https://github.com/OpenUserJs/OpenUserJS.org/wiki) here... preferably animated when possible/needed *(I can do that if it's too much of a learning curve just provide the stills... resolution should be very close to existing images for bandwidth considerations... I'll probably try a post-op optimization anyhow)*... and similar structure to existing documentation... Chrome is the most out of date right now... Average what you can.

I won't be available until next week due to the holiday to assist if necessary... so the PR may not be evaluated until then.

In short with some conditions, absolutely, please, yes :)

**Comment by JoMangee** [comment]
Thanks, I pretty much copies the Opera one - so follows the same format - I've left the images at native res for now - happy for them to be edited.  Will pull it together in next day then.

**Comment by Martii** [comment]
@JoMangee 

`[edgeIcon]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/edge_icon.png "Edge"`

Should render like this for the page:

![][edgeIcon]

[edgeIcon]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/edge_icon.png "Edge"

**Comment by Martii** [comment]
@JoMangee 

So how is it going with this? Thanks.

**Comment by JoMangee** [comment]
@Martii apologies! I tried unsuccessfully to commit or push this - didn't have permissions - but have attached the files I created - in the hope someone else can submit them.


Thanks again.

![commit-jomangee](https://cloud.githubusercontent.com/assets/17394/24161661/f2318e78-0ec9-11e7-99df-35d934fda116.png)
[OpenUserJS.org.wiki.zip](https://github.com/OpenUserJs/OpenUserJS.org/files/859122/OpenUserJS.org.wiki.zip)

**Comment by Martii** [comment]
@JoMangee 
> I tried unsuccessfully to commit or push this - didn't have permissions

That's because for code review in the main repo you push to OpenUserJS/OpenUserJS.org and drop the .md files in at https://github.com/OpenUserJs/OpenUserJS.org/tree/master/views/includes/documents . Any images I'd be happy to review in your zip and get them up there on our CDN of GH's wiki. 

You can use your forks wiki for editing/proofing **or** download GH's "gollum" app to do it locally *(presuming it's still named that)*.

**Comment by Martii** [comment]
Images are up... see example here:

![https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge2.png](https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge2.png)

I was disappointed to find out that the Anniversary Edition wouldn't load TamperMonkey because it's not the correct build. :\

**Comment by Martii** [comment]
Stats for 8 images:
* Originals ~10.9MiB *(11,449,595B)*
* Crushed ~ 335KiB *(343,070B)
* Net savings of ~10.59MiB *(11,106,525B)*

This is part of the reason why the GH wiki is the secondary and not the primary... but the CDN makes it like you are doing it with the attachments of the images... so those can be optimized for bandwidth and the site itself. Courtesy to all of our users is our goal with bandwidth heavily in mind. So thank you for attaching those for review.

**Comment by JoMangee** [comment]
> I was disappointed to find out that the Anniversary Edition (EDIT: meant MSEdge.Win10_RS1.VirtualBox.zip ... need to buy Win10 someday on disc) wouldn't load TamperMonkey because it's not the correct build. :\

Yeah - it is a pain - hence the highlight in the install page:
It works with recent versions of Windows 10 (*14342.0 or higher required*). 

Currently I've got 14393.953 - so it has been working for a while - but had to wait for that - gah at browsers that are tied to OS versions.
